{
  "Header": {
    "Original": "CVE-2022-1679 (A use-after-free flaw was found in the Linux kernel\u0026#8217;s Atheros wi ...)",
    "Line": 51849,
    "ID": "CVE-2022-1679",
    "Description": "(A use-after-free flaw was found in the Linux kernel\u0026#8217;s Atheros wi ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-3173-1 DLA-3131-1}",
      "Line": 51850,
      "Type": "xref",
      "Bugs": [
        "DLA-3173-1",
        "DLA-3131-1"
      ]
    },
    {
      "Original": "- linux 5.19.6-1",
      "Line": 51851,
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "5.19.6-1"
    },
    {
      "Original": "[bullseye] - linux 5.10.140-1",
      "Line": 51852,
      "Type": "package",
      "Release": "bullseye",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "5.10.140-1"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=2084125",
      "Line": 51853,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=2084125"
    },
    {
      "Original": "NOTE: https://lore.kernel.org/lkml/87ilqc7jv9.fsf@kernel.org/t/",
      "Line": 51854,
      "Type": "NOTE",
      "Description": "https://lore.kernel.org/lkml/87ilqc7jv9.fsf@kernel.org/t/"
    }
  ]
}